2200029/4.py

```diff
@@ -0,0 +1,18 @@
+import sys
+sys.stdin = open("input (5).txt", "r")
+
+T = int(input())
+for t in range(1, T + 1):
+    string = input()
+    mirror = {"b": "d", "d": "b", "p": "q", "q": "p"}
+    new_list = []
+    for alpha in string:
+        new_list.append(mirror[alpha])
+        result_list = new_list[::-1]
```

Comment:
딕셔너리 활용 아주 좋습니다! 👍 
```suggestion
    result_list = new_list[::-1]
```
`result_list`는 `for` 문 밖에서 한 번만 구해도 될 것 같습니다!